doc/src/manual/modules.md

```diff
@@ -386,19 +386,24 @@ julia> module ParentModule
        export add_D # export it from ParentModule too
        module SubB
        import ..SubA: add_D # relative path for a “sibling” module
+       import ParentModule.SubA: add_D # equivalent to the previous import
```

Comment:
This failed because the root module here was `Main`. I pushed a commit that allows one to treat a module with Main as parent as a root module which makes this pass but feel free to change things up however you feel fit.

Reply:
But maybe this should only work in packages?

Reply:
I think your approach is good

Reply:
This actually doesn't work to fix the doctest because the code is executed in a separate module so the parent of the package module will in fact not be Main..

Reply:
alright, I just turned this line into an informative comment instead and dropped the change to Main